mystery_word.py

```diff
@@ -0,0 +1,129 @@
+import random
+
+
+def new_word():
```

Comment:
I'd definitely break this into several functions -- one to get the words, one to clean them, and one to pick a word. The way it is, it'll be hard to test.